Sources/Plot/API/HTMLElements.swift

```diff
@@ -426,6 +426,17 @@ public extension Node where Context == HTML.DescriptionListContext {
     }
 }
 
+public extension Node where Context == HTML.DescriptionListContext {
+    /// The `<div>` element wraps a group, that is part of a term-description group in a description list (`<dl>` element).
+    ///
+    /// This is allowed according to the HTML spec: // https://html.spec.whatwg.org/multipage/grouping-content.html#the-dl-element
+    ///
+    /// - parameter nodes: The element's attributes and child elements (`<dl>` or `<dd>` elements).
+    static func dlDiv(_ nodes: Node<HTML.DescriptionListContext>...) -> Node {
```

Comment:
Not quite sure why we can't use `div` here? You mentioned that it being a requirement of `ElementDefinition`, but I don't really understand why that has something to do with this extension? Mind clarifying?

Reply:
Thanks for reviewing this - I cannot remember what error I was getting related to `ElementDefinition` while working on this. I will get back to this if I see or remember the situation.

Reply:
Please wait on this pull request. This is what I get when I changed from `dlDiv` to `div`:

> Type 'ElementDefinitions.Div' does not conform to protocol 'ElementDefinition'

I will see what can be done without adding a new method.

Reply:
Hi John, I'm not sure how to accomplish this without a defining a new function for the context.

What I cannot understand is, if I add `Plot` as a dependency and the following in my app, it works as expected; 
```swift
public extension Node where Context == HTML.DescriptionListContext {

    // The `<div>` element wraps a group, that is part of a term-description group in a description list (`<dl>` element).
    //
    // This is allowed according to the HTML spec: // https://html.spec.whatwg.org/multipage/grouping-content.html#the-dl-element
    
    /// - parameter nodes: The element's attributes and child elements (`<dl>` or `<dd>` elements).
    static func div(_ nodes: Node<HTML.DescriptionListContext>...) -> Node {
        .element(named: "div", nodes: nodes)
    }
}
```
but  when I try to add this to `Plot`, it fails because, `Node.div` becomes ambiguous in

```swift
public enum Div: ElementDefinition { public static var wrapper = Node.div }
```




Since adding it inside the app works for my purposes (I would love to know why though), I can scrap this pull request.

Reply:
Thanks for the update, @nkalvi! Let me look into this a bit later in the week, and I'll let you know what I'll find 👍